gef.py

```diff
@@ -1586,9 +1586,8 @@ def flag_register_to_human(self, val=None):
         return flags_to_human(val, self.flags_table)
 
     def is_conditional_branch(self, insn):
-        branch_mnemos = {"beq", "bne", "bleq", "blt", "bgt", "bgez", "bvs", "bvc",
-                         "jeq", "jne", "jleq", "jlt", "jgt", "jgez", "jvs", "jvc"}
-        return insn.mnemonic in branch_mnemos
+        conditions = {"eq", "ne", "lt", "le", "gt", "ge", "vs", "vc", "mi", "pl", "hi", "ls"}
```

Comment:
1) This doesn't take into account thumb state. If in thumb, `cbz` and `cbnz` should be considered. http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0204g/Cjaghefc.html
2) nit: `cs`, `lo`, and maybe `al`? http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0489e/CEGBHJCJ.html

Reply:
For those I think we need to test what they look like.
For example, `lo` and `cc` are the same thing, but which does GDB choose to represent? We'd need to test this.
`al` is similar: It always jumps, but does GDB show `al` in the mnemonic?

If you'd like to add these, that'd be great. What we really need is some hand written assembly with all of these branches, and then we can 1. test that they work as expected and 2. see how gdb represents them.

If you'd like to open a new PR that's based on this branch and add your own commit (hacktoberfest!), then I'd happily take a look.

Reply:
> What we really need is some hand written assembly with all of these branches

Yeah, all of these corner cases kind of require that.